session/src/test/scala/akka/stream/alpakka/cassandra/scaladsl/CassandraSessionPerformanceSpec.scala

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.cassandra.scaladsl
+
+import scala.concurrent.Await
+
+import akka.actor.ActorSystem
+import akka.event.Logging
+import akka.stream.alpakka.cassandra.{ CassandraSessionSettings, CassandraWriteSettings }
+import akka.stream.scaladsl.{ Sink, Source }
+import akka.stream.testkit.scaladsl.StreamTestKit.assertAllStagesStopped
+import scala.concurrent.duration._
+
+final class CassandraSessionPerformanceSpec extends CassandraSpecBase(ActorSystem("CassandraSessionPerformanceSpec")) {
+
+  val log = Logging(system, this.getClass)
+
+  val data = 1 to 5 * 1000 * 1000
```

Comment:
before merging we can reduce this to avoid running too long/much on travis, with a comment saying that it should be increased for serious testing